linux/network_device_ops/suse_network_conf.j2

```diff
@@ -1,8 +1,8 @@
 NAME='{{ new_network_adapter }}'
 STARTMODE='auto'
-{% if adapter_type != 'pvrdma' %}
+{% if new_nic_ipv4_method == 'dhcp' %}
 BOOTPROTO='dhcp'
-{% else %}
+{% elif new_nic_ipv4_method == 'static' %}
```

Comment:
`new_nic_ipv4_method` can only be `dhcp` or `static`. So here can directly use `else`.

Reply:
Changed, thanks.